roles/ansible-test-splitter/files/split_targets.py

```diff
@@ -28,9 +27,9 @@
         continue
     if not aliases.is_file():
         continue
-    if targets_from_cli and target.name not in targets_from_cli:
-        continue
     lines = aliases.read_text().split("\n")
+    if targets_from_cli and not {*lines, target.name} & set(targets_from_cli):
```

Comment:
Can you do the same for the first one :-)?

Reply:
No, the `set()` function only accepts a single argument. This is why I used the other syntax. I could create a new list and use that, I guess, though.

Reply:
ah I see.